.dagger/docs.go

```diff
@@ -198,6 +199,89 @@ func (d Docs) GenerateSchemaReference() *dagger.Directory {
 	return dag.Directory().WithFile(generatedAPIReferencePath, generatedHTML)
 }
 
+func (d Docs) GenerateReferencePages(ctx context.Context) (*dagger.File, error) {
+	dir := d.GenerateConfigSchemas()
+
+	contents, err := dir.File(generatedDaggerJSONSchemaPath).Contents(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("error reading contents: %v", err)
+	}
+
+	var schema struct {
```

Comment:
Could instead use `"github.com/invopop/jsonschema"` here? No objection to pulling it in as a dependency.

Reply:
I did give that a go and it was a little more tricky. I decided the just make a custom struct with the fields we need. Want me to update the schema to use that package?

Reply:
As-is is fine, just a suggestion in case you wanted to!